pkg/router/template/router.go

```diff
@@ -922,16 +932,27 @@ func (r *templateRouter) createServiceAliasConfig(route *routev1.Route, backendK
 			activeServiceUnits++
 		}
 	}
+	var httpResponseHeadersList []HTTPHeader
+	var httpRequestHeadersList []HTTPHeader
+	if route.Spec.HTTPHeaders.Actions.Response != nil && len(route.Spec.HTTPHeaders.Actions.Response) != 0 {
```

Comment:
What happens if `route.Spec.HTTPHeaders.Actions` was omitted?

Reply:
It can't be omitted the API is not structured that way. I already told you it is not a pointer.

Reply:
```suggestion
	if route.Spec.HTTPHeaders.Actions.Response != "" {
```

Reply:
response is an array